### ISSUE TITLE
How can i enable picture in picture in player

### ISSUE THREAD

**Issue by Najamus** [changsanjiang/SJVideoPlayer#497]
Basically i wanted to add Picture in Picture functionality in player i tried but didn't found a solution how to enable it while using this library.

Thanks & Regards

**Comment by changsanjiang** [comment]
@Najamus 
```Objective-C
// start pip
[_player.playbackController startPictureInPicture];

// stop
// [_player.playbackController stopPictureInPicture];

// pip methods
// - (BOOL)isPictureInPictureSupported API_AVAILABLE(ios(14.0));
// @property (nonatomic) BOOL requiresLinearPlaybackInPictureInPicture API_AVAILABLE(ios(14.0));
// @property (nonatomic, readonly) SJPictureInPictureStatus pictureInPictureStatus API_AVAILABLE(ios(14.0));
// @property (nonatomic, copy, nullable) void(^restoreUserInterfaceForPictureInPictureStop)(id<SJVideoPlayerPlaybackController> controller, void(^completionHandler)(BOOL restored));
// - (void)startPictureInPicture API_AVAILABLE(ios(14.0));
// - (void)stopPictureInPicture API_AVAILABLE(ios(14.0));
```

**Comment by Najamus** [comment]
@changsanjiang thanks brother but i need to confirm one thing how to add picture in picture icon in player currently there is no icon is showing related to pip.

Thanks For Your Reply.

**Comment by changsanjiang** [comment]
@Najamus you can add a custom pip button to player or using SJVideoPlayer. (there are no such buttons on the SJBaseVideoPlayer);

```Objective-C
        startImage = [AVPictureInPictureController.pictureInPictureButtonStartImage imageWithTintColor:UIColor.whiteColor];
        stopImage = [AVPictureInPictureController.pictureInPictureButtonStopImage imageWithTintColor:UIColor.whiteColor];
```

**Comment by Najamus** [comment]
im already using SJVideoPlayer, how can i add custom pip button in SJVideoPlayer?

**Comment by Najamus** [comment]
HI Brother Just tell me one thing, im using SJVideoPlayer, when i enabled picture in picture in my project from capabilities now player default show picture in picture icon on top right & its working correctly.

i just need to change that pip icon tint color currently it adopt application accent/tint color but i wnat to change it with white color.

Thanks & Regards 
Najam

**Comment by changsanjiang** [comment]
@Najamus 

```swift
    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
        
        SJVideoPlayer.updateResources{ res in
//            res.progressThumbSize = 8
//            res.progressTraceColor = UIColor.white
//            res.progressTrackColor = UIColor(white: 0.4, alpha: 1.0)
//            res.progressBufferColor = UIColor(white: 0.3, alpha: 1.0)
            
            if #available(iOS 14.0, *) {
                res.pictureInPictureItemStartImage = AVPictureInPictureController.pictureInPictureButtonStartImage.withTintColor(.red)
                res.pictureInPictureItemStopImage = ...
            }
        }

        // Override point for customization after application launch.
        return true
    }
```

**Comment by trial** [comment]
setting image color seems no effect,
pictureInPictureItemStartImage is still blue and pictureInPictureItemStopImage is still white
